packages/atomic/cypress/integration/facets/numeric-facet/numeric-facet-actions.ts

```diff
@@ -0,0 +1,83 @@
+import {setUpPage} from '../../../utils/setupComponent';
+
+export const numericField = 'size';
+export const numericLabel = 'File Size';
+export const numericRanges: NumericRange[] = [
+  {
+    start: 0,
+    end: 100000,
+  },
+  {
+    start: 100001,
+    end: 1000000,
+  },
+  {
+    start: 1000001,
+    end: 10000000,
+  },
+];
+
+export interface NumericRange {
+  start: number;
+  end: number;
+}
+
+export interface NumericFacetSetupOptions {
+  field: string;
+  label: string;
+  ranges: NumericRange[];
+  attributes: string;
+  executeFirstSearch: boolean;
+}
+
+export function generateDateRangeHtml(numericRanges: NumericRange[] | []) {
+  if (!Array.isArray(numericRanges) || !numericRanges.length) {
```

Comment:
It would be possible to remove this `if` since a) typescript assures an array is passed, and b) mapping over an empty array and joining should return an empty string. Also in the case of an empty array, one could simply not call this function,